### ISSUE TITLE
Export interface

### ISSUE THREAD

**Issue by equinusocio** [rpearce/image-zoom#287]
This issue is a:

- Feature request

## Description

It would be nice if we can import and use the component interface inside ts to merge their property with custom ones. For example I need to wrap the Zoom component inside a custom component and i need to add other properties, but i would like to merge my custom prop with the one from the Zoom, so i don't have to expose them one by one.

**Comment by rpearce** [comment]
Yeah, all of this should be exported. Let me put a PR up and see if you like how it looks

**Comment by rpearce** [comment]
@equinusocio Is this what you're looking for? https://github.com/rpearce/image-zoom/pull/289/commits/8d2ecfa935b1620923b0496647b68d7eaa1f78be

I know you don't need the helpers exported, but I figure there's no harm.

Also, would you consider this change a patch-level change (a fix)?

**Comment by equinusocio** [comment]
I think that shared and reusable components should have heir types exported "by default", so we can consider this as a "fix" for this lack.

**Comment by rpearce** [comment]
Great call on the `Props` naming. I'll amend that when I get a chance today
